### PR TITLE
Upversion to 2.0.10-2

### PR DESCRIPTION
**Pull request by yjian118**
This commit upversions to 2.0.10-2

### CODE REVIEW

File: api/v1/constructors.go
Comment (sselvara1) [inline api/v1/constructors.go:1]: how did it work previously with ParseVolumeGroupInfo ? and now its parseVolumeGroupInfo

File: DEBUGGING.md
Comment (sselvara1) [inline DEBUGGING.md:1]: up version to be done for every commit we are going add here onwards?

Comment (yjian118) [reply DEBUGGING.md:1]: Yes... That's the conclusion we get with the registry admin

File: api/v1/constructors.go
Comment (yjian118) [reply api/v1/constructors.go:1]: Would prefer to start with uppercase if it is a public method. But this change is not included in this pull request

File: DEBUGGING.md
Comment (yjian118) [reply DEBUGGING.md:1]: To be more precise, the registry admin won't do the image tag update and re-push work, so if an image is pushed to the staging area, we can only update the image tag if we want more commits to be included

**Review by ttakenak — APPROVED** [review]

**Pull request merged**